### ISSUE TITLE
latest pro 2.2.1 causes content to vanish [BUG] -> BIG ONE !

### ISSUE THREAD

**Issue by dasigna** [pojome/elementor#6292]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue EXCLUSIVELY exists against the latest stable version of Elementor.

## Description

wtih latest ELpro 2.2.1 the elementor created content on frontend vanishes.
editor doesnt load completely

## Steps to reproduce

just install Pro 2.2.1. and see :)

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Percona Server (GPL), Release '22', Revision 'f62d93c' v5.7.22-22
	PHP Version: 7.2.10
	PHP Max Input Vars: 1500
	PHP Max Post Size: 32M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 4.9.8
	Site URL: https://www.-
	Home URL: https://www.-
	WP Multisite: No
	Max Upload Size: 32 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: de-DE
	Timezone: Europe/Berlin
	Admin Email:
	Debug Mode: Inactive

== Theme ==
	Name: GeneratePress Child
	Version: 1.1
	Author: ar.
	Child Theme: Yes
	Parent Theme Name: GeneratePress
	Parent Theme Version: 2.2.1
	Parent Theme Author: Tom Usborne

== User ==
	Role: administrator
	WP Profile lang: de_DE
	User Agent: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:64.0) Gecko/20100101 Firefox/64.0

== Active Plugins ==
	Admin Columns
		Version: 3.3.1
		Author: AdminColumns.com

	Admin Menu Tree Page View
		Version: 2.7.1
		Author: Pär Thernström

	Advanced Custom Fields
		Version: 5.7.7
		Author: Elliot Condon

	BackWPup
		Version: 3.6.6
		Author: Inpsyde GmbH

	Cache Enabler
		Version: 1.3.1
		Author: KeyCDN

	Custom Archive Titles
		Version: 1.1
		Author: ThemeZee

	Custom Post Type UI
		Version: 1.6.0
		Author: WebDevStudios

	Ele Custom Skin
		Version: 1.0.9
		Author: Liviu Duda

	Elementor
		Version: 2.3.3
		Author: Elementor.com

	Elementor Extras
		Version: 1.9.16
		Author: Namogo

	Elementor Pro
		Version: 2.2.1
		Author: Elementor.com

	Enable Media Replace
		Version: 3.2.7
		Author: ShortPixel

	Force Regenerate Thumbnails
		Version: 2.0.6
		Author: Pedro Elsner

	GP Premium
		Version: 1.7.5
		Author: Tom Usborne

	Physical Custom Upload Dir for Real Media Library
		Version: 1.0.4
		Author: Matthias Günter

	Relevanssi
		Version: 4.1.1.2
		Author: Mikko Saari

	Widget Visibility Time Scheduler
		Version: 5.3.1
		Author: Martin Stehle

	WP Author Slug
		Version: 2
		Author: Konstantin Obenland

	WP Real Media Library
		Version: 4.0.7
		Author: Matthias Günter

== Debug ==
	Errors: 
		TypeError: e is undefined; can't access its "$element" property
			at https://www.-.de/wp-content/plugins/elementor/assets/js/frontend.min.js?ver=2.3.3 - 2:34401
			2018-11-28 14:11 +00:00

		TypeError: setting getter-only property "type"
			at https://www.-.de/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.2.7 - 2:86630
			2018-10-26 13:02 +00:00
			x 6 times

		TypeError: this._parent is undefined; can't access its "errors" property
			at https://www.-.de/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.2.7 - 2:173870
			2018-10-26 08:51 +00:00
			x 16 times

		TypeError: setting getter-only property "type"
			at https://www.-.de/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.2.5 - 2:86630
			2018-10-15 15:55 +00:00
			x 2 times

		TypeError: this.ui.select is undefined
			at https://www.-.de/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.2.5 - 2:13050
			2018-10-10 09:06 +00:00

		TypeError: o.collection is undefined
			at https://www.-.de/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=2.1.9 - 2:48714
			2018-10-09 13:42 +00:00
			x 4 times

		TypeError: setting getter-only property "type"
			at https://www.-.de/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.2.4 - 2:86630
			2018-10-06 18:38 +00:00

</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by formulaclick** [comment]
Same issue here...

**Comment by sermalefico** [comment]
Lo mismo

**Comment by CDiFazio16** [comment]
Same issue, slideshow on my homepage disappeared and unable to access any page through elementor editor since update.

**Comment by sermalefico** [comment]
`TypeError: Cannot read property '$element' of undefined
			at http://www.adrianarcay.com/contenidos/plugins/elementor/assets/js/frontend.min.js?ver=2.3.3 - 2:34417
			2018-11-28 14:37 +00:00
			x 2 times
`

**Comment by TuffG** [comment]
Same here

**Comment by enricoberti** [comment]
Same same!

**Comment by sermalefico** [comment]
change label for BUG

**Comment by dasigna** [comment]
... seems to be fixed with 2.2.2 already -> GREAT!

**Comment by enricoberti** [comment]
awesome! thanks for the prompt fix!

**Comment by jrossmurph** [comment]
> ... seems to be fixed with 2.2.2 already -> GREAT!

Did you get this update from elementor website or were you prompted to update in wordpress? I also have a launch today (couple hours) and am unable to make last minutes changes. eek.

**Comment by sermalefico** [comment]
GReat! but the advanced custom image fields in the Taxonomy templates are still not working. A mistake of yesteryear. Now apear this - - appears above the custo mfields

**Comment by bainternet** [comment]
Issue fixed in 2.2.2
